src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -23,26 +23,22 @@
 	<AppNavigationItem
 		v-click-outside="closeShareMenu"
 		:loading="calendar.loading"
+		:aria-description=" calendar.enabled ? $t('calendar', 'Disable {calendar} calendar', {calendar: calendar.displayName.toLowerCase()}) :
```

Comment:
I would like to propose changing the strings a little bit. Especially in German making the name lowercase will not work well. E.g. a calendar named `Feiertage` (holidays) will produce the string `Deaktiviere feiertage Kalendar`, which does not make sense.
I would propose to use `$t('calendar', 'Disable calendar "{calendar}"', {calendar: calendar.displayName})` which will give `Deaktiviere Kalendar "Feiertage"`.